### PR TITLE
feat(service): expose tech_emails in Service

### PR DESCRIPTION
**Pull request by jeff-held-aiven**

### CODE REVIEW

File: service_test.go [hunk 1]
```diff
@@ -9,372 +9,239 @@ import (
...
 
-func setupServiceTestCase(t *testing.T) (*Client, func(t *testing.T)) {
-	t.Log("setup Service test case")
-
-	const (
-		UserName     = "test@aiven.io"
-		UserPassword = "testabcd"
-		AccessToken  = "some-random-token"
-	)
+var servicesHandler = &ServicesHandler{
```
Comment (Serpentiel) [inline service_test.go:12]: I'm not sure if this is a good idea, with this approach it's possible to use `servicesHandler` in other tests as well, which might be unexpected, WDYT?

File: service.go [hunk 2]
```diff
@@ -230,6 +231,7 @@ type (
 		ServiceIntegrations   []NewServiceIntegration `json:"service_integrations"`
 		DiskSpaceMB           int                     `json:"disk_space_mb,omitempty"`
 		StaticIPs             []string                `json:"static_ips,omitempty"`
+		TechnicalEmails       *[]*ContactEmail        `json:"tech_emails,omitempty"`
 	}
```
Comment (byashimov) [inline service.go:235]: Hey. Nil is not a value here, so no need to be a pointer for the slice.

File: service_test.go [hunk 3]
```diff
@@ -9,372 +9,239 @@ import (
...
+		got, err := servicesHandler.List(ctx, "test-pr")
+
+		if err != nil {
+			t.Errorf("Unexpected error was returned when listing services: %v", err)
+			return
+		}
+		if !reflect.DeepEqual(got, []*Service{}) {
+			t.Errorf("Got %v, expected %v", got, []*Service{})
+		}
+	})
```
Comment (byashimov) [inline service_test.go:246]: Please do not refactor the client. It will be replaced soon with code-generated one.

**Review by byashimov — CHANGES_REQUESTED** [review]

**Review by byashimov — APPROVED** [review]
lgtm

**Pull request merged**